safetrace-sdk/Services/Session/UserSession.swift

```diff
@@ -14,11 +14,11 @@ struct AuthData: Codable {
 private let authTokenKeychainIdentifier = "UserToken"
 private let userIDKeychainIdentifier = "UserId"
 
-#if INTERNAL
-private let keychainAppIdentifier = "L5262XM8UA.org.ctzn.safetrace-app-dev"
+#if DEV
```

Comment:
Hmm somehow `DEV` isn't recognized by the compiler flags. Will investigate more

Reply:
Ah it's because the flag was set in Safetrace-app, and not accessible in Safetrace-sdk